media/com_banners/js/admin-banner-edit.es6.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @copyright  Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+Joomla = window.Joomla || {};
```

Comment:
This can be moved inside the IIFE like: `window.Joomla = window.Joomla || {};`

OR a better approach, since we're require the API to be present in order this script to work we can:
```js
if (!window.Joomla) {
  throw new Error('API is required, blabla');
}
```